articles/azure/azs-how-create-availability-set.md

```diff
@@ -0,0 +1,166 @@
+---
+title: How to create a Availability Set | UKCloud Ltd
+description: Creating a Availability Set in UKCloud for Microsoft Azure
```

Comment:
Describes how to create an availability set in UKCloud for Microsoft Azure

Reply:
done